### PR TITLE
Further Training Improvements

### PR DESCRIPTION
**Pull request by nrichers**
### New intent

- Establish a slightly different naming convention for course paths (role > course name)
- Fix the 404 Beck identified
- Experiment:
   - Further `.preview` improvements:
      - Support external URLs — ec9f2c5b409f5d6917cc897cf6a0a1ab7df4a9a9 (different branch)
      - Support optional width & height parameters — 6ccd6ac6ef703c1b2b9cf493dfc23d1dd8e07dd7 (different branch)

### Original intent — was mostly included in Beck's original PR 

This PR modifies @validbeck's #501 to create a more home page-like feel for our training content:

- Change `title` to `pagetitle`
- Add a hero banner for training 
- Switch column widths to hide left navtree and space out sections
- Add some bespoke CSS to support the new look

image

### CODE REVIEW

File: site/training/assets/training.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    padding-bottom: 20px;
+    padding-left: 30px;
+    padding-right: 40px;
+  }
+  
+.attn h2{
+    color: #042426;
+  }
+
+.preview {
```
Comment (nrichers) [inline site/training/assets/training.css:113]: Need to investigate but I did something to break my `.preview` extension ... It affects all course registration pages. Example:

image

Comment (nrichers) [reply site/training/assets/training.css:113]: It has to be an issue in this PR as Beck's initial PR that my branch is based on does not exhibit this behaviour. But here's a data point: it looks like we're still showing the `.qmd` file in the output when we should be referencing the `.html`:

image

**Comment by validbeck** [comment]
@nrichers When you shuffle the folder names around, can you make sure the example is updated in the training section of the style guide as well? My silly butt missed it on the 1st round...

Screenshot 2024-11-04 at 5 39 32 PM

**Comment by nrichers** [comment]
> @nrichers When you shuffle the folder names around, can you make sure the example is updated in the training section of the style guide as well?

Yup! Not a problem at all.

**Comment by nrichers** [comment]
This PR has been replaced by https://github.com/validmind/documentation/pull/509, closing.

**Pull request closed without merging**